rllib/agents/ddpg/ddpg.py

```diff
@@ -106,10 +106,21 @@
     "prioritized_replay_eps": 1e-6,
     # Whether to LZ4 compress observations
     "compress_observations": False,
-    # If set, this will fix the ratio of replayed from a buffer and learned on
-    # timesteps to sampled from an environment and stored in the replay buffer
-    # timesteps. Otherwise, the replay will proceed at the native ratio
-    # determined by (train_batch_size / rollout_fragment_length).
+
+    # The intensity with which to update the model (vs collecting samples from
+    # the env). If None, uses the "natural" value of:
+    # `train_batch_size` / (`rollout_fragment_length` x `num_workers` x
+    # `num_envs_per_worker`).
+    # If provided, will make sure that the ratio between ts inserted into and
+    # sampled from the buffer matches the given value.
+    # Example:
+    #   training_intensity=1000.0
+    #   train_batch_size=250 rollout_fragment_length=1
+    #   num_workers=1 (or 0) num_envs_per_worker=1
+    #   -> natural value = 250 / 1 = 250.0
+    #   -> will make sure that replay+train op will be executed 4x as
+    #      often as rollout+insert op (4 * 250 = 1000).
+    # See: rllib/agents/dqn/dqn.py::calculate_rr_weights for further details.
     "training_intensity": None,
```

Comment:
Don't really like this hyperparameter. It needs to be more specific such as update_frequency

Reply:
True :) , that's why it's None by default. Just updated the comment b/c some users had asked for this wrt SAC.
The problem I think is the cross dependency with `rollout_fragment_length`, which makes it hard to just allow setting a `update_frequency`.

True, probably the best would be to do:
`rollout_fragment_length`: determines the number of samples in a "sample+insert" step (which is already does today).
`sample_insert_vs_train_ratio`: determines, how many samples+inserts we do before we do a train update. This can also be < 1.0 then. Thoughts?

Reply:
I think the second option is good. Its very specific